src/components/Form/FormBuilder.js

```diff
@@ -19,11 +18,9 @@ function FormBuilder({
   onSubmit,
   title,
 }) {
-  const { t } = useTranslation();
-
   return (
     <Form
-      buttonLabel={buttonLabel || t('generic.form.submitLabel')}
```

Comment:
just wondering - why was this or value removed?

Reply:
Totally!
The `Form` component has a new default behavior for Submit button. Setting the `buttonLabel` overrides that default behavior, so taking this label out of the Formbuilder allows the `buttonLabel` override to tunnel down into the Form.

Reply:
Here's the logic for the Submit button label:
https://github.com/helpsupply/helpsupply/blob/3c19497980883240cb5c6ae3cca182a9ff409755/src/components/Form/index.js#L48